package.json

```diff
@@ -51,6 +52,7 @@
     "dedent": "^0.7.0",
     "envalid": "^7.3.1",
     "got": "^11.8.6",
+    "js-yaml": "^4.1.0",
```

Comment:
We already have `yaml` installed; would it make sense to use `yaml`?

If you feel `js-yaml` is an improvement, feel free to open a separate PR on that 👍